README.md

```diff
@@ -2,54 +2,76 @@
+Stager is a Flutter development tool that allows you to run small portions of your app as individual Flutter apps. This lets you:
 
-Stager can accelerate your development workflow for widgets that:
+- Focus your development on a single widget or flow – no more clicking through multiple screens or setting external feature flags to reach the page you're working on.
+- Ensure your UI works in *all* cases, including:
+  - Light and Dark mode
+  - Small or large text sizes
+  - Different viewport sizes
+  - Different device types
+  - Loading, empty, error, and normal states
+- Show all of this to your designers to make sure your app is pixel-perfect.
 
-- Have multiple states (empty, error, loading, etc.) that would otherwise require code changes to trigger.
-- Are cumbersome to navigate to.
-- Are hidden behind a feature flag.
-- Behave differently based on external state (e.g., the type of currently logged-in user).
+## Demo
+
+![example app demo](https://user-images.githubusercontent.com/581764/219502623-bfe44091-a582-460e-b97d-e786bc614a8c.gif)
+
+The example included in this repo demonstrates how Stager can be used in the context of a simple Twitter-like app that displays a feed of posts and includes detail pages for posts and users.
+
+Stager uses Scenes (see the [Concepts](#concepts) section below) that you define to generate small Flutter apps. To run the Stager apps included in the example, start by moving to the `example` directory and fetching the app's dependencies:
+
+```bash
+cd example
+flutter pub get
+```
+
+You can then run the indivdual Stager apps with the following commands:
+
+**Posts List**
+```bash
+flutter run -t lib/pages/posts_list/posts_list_page_scenes.stager_app.g.dart
+```
+
+**User Detail**
+```bash
+flutter run -t lib/pages/user_detail/user_detail_page_scenes.stager_app.g.dart
+```
 
-A Stager app for a ListView displaying forum-style posts:
+**Post Detail**
+```bash
+flutter run -t lib/pages/post_detail/post_detail_page_scenes.stager_app.g.dart
+```
 
-![example app demo](https://user-images.githubusercontent.com/581764/181614468-cbb89cbe-d16a-44bf-831e-47139ce3a7c0.gif)
+To get an idea of how these Scenes fit together, you can also run the main app by executing `flutter run` from the `example` directory.
 
 ## Concepts
 
 ### Scene
 
-The most important class in Stager is the Scene class. A Scene is a simple, self-contained unit of UI. Scenes make it easy to focus on a single widget or page to greatly increase development velocity by isolating them from the rest of your app and allowing fine control of dependencies.
-
-A Scene has three parts:
+A Scene is a simple, self-contained unit of UI, and is the most important idea in Stager. Scenes make it easy to focus on a single widget or page to greatly increase development velocity by isolating them from the rest of your app and allowing fine control of dependencies.
```

Comment:
"allowing fine control of dependencies." sounds a bit vague to me. What about:
 
"isolating them from the rest of your app and avoiding the need to build all transitive dependencies"
"isolating them from dependencies that are slow to build or difficult to configure".
"isolating them from the rest of your app"